test/new.jl

```diff
@@ -2417,7 +2417,8 @@ end
             Pkg.add(Pkg.PackageSpec(uuid=exuuid, version=v"0.5.3"))
         catch e
             @test e isa ResolverError
-            @test occursin("possible versions are: 0.5.1 or uninstalled", e.msg)
+            # `\S*` in regex below will allow for ANSI color escape codes in the logs
+            @test occursin(r"possible versions are: \S*0\.5\.1\S* or uninstalled", e.msg)
```

Comment:
```suggestion
            @test occursin(r"possible versions are: [^\d\s]*0\.5\.1[^\d\s]* or uninstalled", e.msg)
```
This should ensure that the leading/trailing "junk" is not digits and not whitespace.

Reply:
That "junk" will contain digits.
It is something like: `"\e[34m` and `\e[39m"`

Reply:
CI agrees that it will contain digits and failed.
I have reverted your commit.